docker/dev/ubuntu/Dockerfile

```diff
@@ -0,0 +1,35 @@
+# Develop Orion Context Broker Ubuntu-Latest
+FROM ubuntu:latest
```

Comment:
I agree with Dani. But this image I understand is for development of Orion, not to deploy an Orion to use. I guess this is for folks who want to use other OS than CentOS for development. That's why I think the CentOS one should be preferred.

Then, for deploying Orion or just trying it this image wouldn't make much sense.